barman/command_wrappers.py

```diff
@@ -1150,6 +1150,71 @@ def __init__(
             self.args += args
 
 
+class PgCombineBackup(PostgreSQLClient):
+    """
+    Wrapper class for the pg_combinebackup system command
+    """
+
+    COMMAND_ALTERNATIVES = ["pg_combinebackup"]
+
+    def __init__(
+        self,
+        destination,
+        command,
+        tbs_mapping=None,
+        connection=None,
+        version=None,
+        app_name=None,
+        check=True,
+        args=None,
+        **kwargs
+    ):
+        """
+        Constructor
+
+        :param PostgreSQL connection: an object representing
+          a database connection
+        :param str destination: destination directory path
+        :param str command: the command to use
+        :param Version version: the command version
+        :param str app_name: the application name to use for the connection
+        :param str bwlimit: bandwidth limit for pg_basebackup
+        :param Dict[str, str] tbs_mapping: used for tablespace
+        :param bool immediate: fast checkpoint identifier for pg_basebackup
+        :param bool check: check if the return value is in the list of
+          allowed values of the Command obj
+        :param barman.compression.PgBaseBackupCompression compression:
+          the pg_basebackup compression options used for this backup
+        :param str parent_backup_manifest_path:
+          the path to a backup_manifest file from a previous backup which can
+          be used to perform an incremental backup
+        :param List[str] args: additional arguments
```

Comment:
```suggestion
        :param str destination: destination directory path
        :param str command: the command to use
        :param None|Dict[str, str] tbs_mapping: used for tablespace
        :param PostgreSQL connection: an object representing
          a database connection
        :param Version version: the command version
        :param str app_name: the application name to use for the connection
        :param bool check: check if the return value is in the list of
          allowed values of the :class:`Command` obj
        :param None|List[str] args: additional arguments
```

The docstring had arguments in the wrong worder, or arguments from other commands (probably result of a copy'n' paste).

This is a suggestion to fix it.